packages/manager/modules/pci/src/components/new-region-banner/controller.js

```diff
@@ -0,0 +1,18 @@
+const BETA_BANNER_END_DATE = '2021-09-15';
+
+export default class NewRegionBannerComponentController {
+  /* @ngInject */
+  constructor($state) {
+    this.$state = $state;
+  }
+
+  $onInit() {
+    this.isEndDateToDisplayBetaBanner = moment().isAfter(BETA_BANNER_END_DATE);
```

Comment:
Maybe we could use feature flipping for that (if date change, no need to update this part).
And we could also rename it to something like `displayBetaBanner`

Reply:
Ah yes it could be a good idea.
For me i am not sure we need to do that into the 2API side because it's a temporary component.
After the end date i will remove it.

If it's required i can do it but at the end i will remove the component.
please let me know your point of view, thanks.

Reply:
I got my reponse

Reply:
done